kafka-java-liberty-sample/src/main/java/com/eventstreams/samples/servlet/KafkaServlet.java

```diff
@@ -1,17 +1,107 @@
- *  See the License for the specific language governing permissions and
- *  limitations under the License.
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
+ * <p>
+ * Licensed Materials - Property of IBM
+ * (c) Copyright IBM Corp. 2016
```

Comment:
Again is this needed?

Reply:
This was on the original codebase - these changes, I think, are formatting related. If you have an intellij or eclipse formatter I can reformat and resubmit the pr.